packages/web/src/server/index.ts

```diff
@@ -98,12 +107,26 @@ export async function track(
 
     const url = new URL(origin);
 
+    let flagsToReport: Record<string, unknown> = {};
+    const allFlags = requestContext?.flags?.getValues();
+    if (options?.flags && allFlags) {
```

Comment:
Does this mean we made the decision that flags will only get tracked when specified explicitly? No auto-tracking of all known flags?

I don't have an opinion here yet, just curious if this was a conscious decision

Reply:
No, check the `else if` closure here. We'll default to all flags and let the API decide by default. The `options.flags` only provide a way to be more specific when needed.

Reply:
Oh, totally missed that!

If a user wanted to opt out of tracking any flags by default, and only opt certain `track` calls into tracking some flags, would this work already? I guess we're adding the global setting later on?